wikichange/assets/content.css

```diff
@@ -148,20 +148,7 @@ button:disabled.highlightButton {
 #belowGraphDiv {
   padding-top: 2%;
 }
-#loader {
-    padding-top: 3px; 
-    border-width: 5px; 
-    border-style: solid; 
-    border-color: rgb(52, 152, 219) white white; 
-    border-image: initial; 
-    border-radius: 50%; 
-    width: 15px; height: 15px;
-    position: absolute; 
-    margin-top: 6.5px;
-    margin-left: 40%; 
-    display: none; 
-    animation: 2s linear 0s infinite normal none running spin;
-}
+
```

Comment:
Should add `margin-left: 4px;` to the `button.highlightButton` above to add space between the button and date selector
<img width="334" alt="image" src="https://user-images.githubusercontent.com/55929961/218296921-882dc7f2-5684-4b57-ae89-5c0ef7185b08.png">